packages/feedback-service/src/schema.ts

```diff
@@ -5,10 +5,10 @@ const FeedbackSchema: Schema = new Schema({
   experience: String,
   ticketID: String,
   spa: String,
-  createdAt: { type: Date, default: Date.now },
-  createdBy: { type: String, ref: 'User' },
-  modifiedAt: Date,
-  modifiedBy: { type: String, ref: 'User' },
+  createdOn: { type: Date, default: Date.now },
+  createdBy: String,
+  updatedOn: { type: Date, default: Date.now },
+  modifiedBy: String,
```

Comment:
As discussed yesterday, replace all `modifiedAt` and `modifiedBy` with `updatedOn` and `updatedBy`

```suggestion
  updatedBy: String,
```